### PR TITLE
Enable commitlint

### PR DESCRIPTION
**Pull request by fdmota**
Ensure that commits follow the right message format which is important for later versioning.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -12,8 +12,6 @@
         "packages/*"
       ],
       "dependencies": {
-        "@commitlint/cli": "^16.2.1",
```
Comment (georgeroman) [inline package-lock.json:15]: Can we remove `package-lock.json`? The convention is to use `yarn` instead of `npm` and so the `npm` lock file is redundant.

Comment (fdmota) [reply package-lock.json:15]: I can, but the package-lock.json already exists in the repo. Should I remove it?

Comment (georgeroman) [reply package-lock.json:15]: I think so, nothing should depend on that. @nofir @ipeleg do you agree?

Comment (nofir) [reply package-lock.json:15]: yeah, i think it can be removed.

Comment (fdmota) [reply package-lock.json]: removed

**Review by georgeroman — APPROVED** [review]

**Pull request merged**